doc/rst/language/spec/locales.rst

```diff
@@ -239,7 +239,8 @@ directly.
    variable ``r`` is a record and has value semantics. It exists on
    ``Locales(1)`` even though it is assigned a value on a remote locale.
 
-Global (non-distributed) constants are replicated across all locales.
+Module-scope constants that are not distributed in nature are
+replicated across all locales.
```

Comment:
This makes me wonder what constants are "distributed in nature". Can I make one myself?

Reply:
`const A: [MyDistDom] real = 0.0;`